lib/github-markup.rb

```diff
@@ -1,6 +1,6 @@
 module GitHub
   module Markup
-    VERSION = '3.0.1'
+    VERSION = '3.0.2'
```

Comment:
Not sure if this is appropriate to do here or if we want to roll out a release as a separate PR. Might also be worth more than a patch version; maybe 3.1.0? cc @kivikakk @lildude